CONTRIBUTING.md

```diff
@@ -115,6 +116,27 @@ the project with Local File System input.
 resources/sample_md_reflections/lab1, resources/sample_md_reflections/lab2, resources/sample_md_reflections/lab3
 ```
 
+#### Missing Module Gensim or gensim.summarization With Windows
+
+If you run streamlit_web.py and are met with a `No module named Gensim` or similar
+error and are running windows, You can fix this problem by installing Microsoft Visual
+C++ 14.0+. You can do this by going to the [Visual Studio 2019 Downloads](https://visualstudio.microsoft.com/downloads/),
+scrolling down and expanding the `Tools for Visual Studio 2019` and clicking the download link for
+`Build tools for Visual Studio 2019`. 
```

Comment:
Instead of describing the error, maybe just say that GatorMiner partially depends on `gensim`, and `gensim` requires Microsoft Visual C++ 14.0+. That way, developers can check whether their version is compatible beforehand, instead of running the tool and unsure about whether they would encounter an error. 

Essentially, the issue is that `gensim` can't be successfully installed without the required Visual C++ version. `No module named Gensim` is just a generic error output message, which could be misleading to some extent. That's why I don't think it's necessary to mention it.